server/src/main/java/org/elasticsearch/index/fielddata/plain/AbstractIndexOrdinalsFieldData.java

```diff
@@ -40,13 +41,13 @@
 
 public abstract class AbstractIndexOrdinalsFieldData extends AbstractIndexFieldData<LeafOrdinalsFieldData>
         implements IndexOrdinalsFieldData {
+    private static final Logger logger = LogManager.getLogger(AbstractBinaryDVLeafFieldData.class);
```

Comment:
This is the one change that I believe I'm actually making in this PR: it'll use the logger for this class rather than its subclasses when loading ordinals.